CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `check-deps` command to validate and update dependency bump changelog entries ([#186](https://github.com/MetaMask/create-release-branch/pull/186))
+  - Automatically detects dependency version changes from git diffs
+  - Validates changelog entries with exact version matching (catches stale entries)
+  - Auto-updates changelogs with `--fix` flag, preserving PR history
+  - Detects package releases and validates/updates in correct changelog section (Unreleased vs specific version)
+  - Smart PR concatenation when same dependency bumped multiple times
+  - Usage: `yarn check-dependency-bumps --fix --pr <number>`
```

Comment:
This is how we'd call the script from _other_ repos? How is that possible when this isn't registered in `package.json` as a `bin` script?

Reply:
Oh I see, didn't see this comment until now. Maybe this should be moved back into draft then?

Reply:
No worries, I will move it back to draft and apply all the suggestions you mentioned here. They are also relevant for auto-changelog, since much of the logic is moved.
Thank you for the review!